include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -1110,34 +1110,35 @@ namespace VectorTools
   {
     const unsigned int fe_degree = matrix_free.get_dof_handler().get_fe().degree;
 
+    (void)n_q_points_1d;
     Assert (fe_degree+1 == n_q_points_1d,
             ExcNotImplemented());
 
     switch (fe_degree)
       {
       case 1:
-        project_parallel<dim, VectorType,1,2> (matrix_free,constraints,func,vec_result);
+        project_parallel<dim, VectorType, dim, 2> (matrix_free,constraints,func,vec_result);
```

Comment:
btw, what are the issues (apart from warnings)? tests run fine for me locally...